components/style/properties/properties.mako.rs

```diff
@@ -552,6 +462,11 @@ impl LonghandId {
         }
     }
 
+    fn inherited(&self) -> bool {
```

Comment:
I think we could try to use this more in a few places, I think it could reduce a bit of code size here and there too.